src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1249,6 +1256,794 @@ subroutine eigencalving(domain, err)
+      real (kind=RKIND), pointer ::  &
+           config_ice_density,            & ! ice density
+           config_ocean_density             ! ocean density
+
+      real (kind=RKIND), save :: rhoi    ! ice density (kg m^{-3}), copied from config_ice_density
+      real (kind=RKIND), save :: rhoo    ! ocean density (kg m^{-3}), copied from config_ocean_density
+
+      type (field1dReal), pointer :: meanFlowParamAField
+      real, dimension(:), pointer :: meanFlowParamA
+      !type (field1dReal), pointer :: viscField
+      !real, dimension(:), pointer :: visc
+      type (field1dReal), pointer :: sigmayxField
+      real, dimension(:), pointer :: sigmayx
+      type (field1dReal), pointer :: sigmaxyField
+      real, dimension(:), pointer :: sigmaxy
+      type (field1dReal), pointer :: sigmaxxField
+      real, dimension(:), pointer :: sigmaxx
+      type (field1dReal), pointer :: sigmayyField
+      real, dimension(:), pointer :: sigmayy
+      type (field1dReal), pointer :: sigmaminField
+      real, dimension(:), pointer :: sigmamin
+      !type (field1dReal), pointer :: sigmamaxField
+      !real, dimension(:), pointer :: sigmamax
+      type (field1dReal), pointer :: alphaField
+      real, dimension(:), pointer :: alpha
+      !type (field1dReal), pointer :: epsmaxField
+      !real, dimension(:), pointer :: epsmax
+      !type (field1dReal), pointer :: nstarField
+      !real, dimension(:), pointer :: nstar
+      !type (field1dReal), pointer :: szeroField
+      !real, dimension(:), pointer :: szero
+      !type (field1dReal), pointer :: sourceField
+      !real, dimension(:), pointer :: source
+      !type (field1dReal), pointer :: ddamagedtField
+      !real, dimension(:), pointer :: ddamagedt
+      type (field1dReal), pointer :: forceField
+      real, dimension(:), pointer :: force
+      type (field1dReal), pointer :: meanTField
+      real, dimension(:), pointer :: meanT
+
+      ! TZ: we can comment out the scratch field as above and redefine it in Registry.xml if we want to see it in the output nc file
+
+      integer, dimension(:), pointer :: cellMask
+      type (field1dInteger), pointer :: calvingFrontMaskField
+      integer, dimension(:), pointer :: calvingFrontMask
+      real (kind=RKIND), pointer :: deltat, daysSinceStart !< time step (s)
+      integer, dimension(:), pointer :: nEdgesOnCell ! number of cells that border each cell
+      integer, dimension(:,:), pointer :: cellsOnCell ! list of cells that neighbor each cell
+      integer, dimension(:,:), pointer :: edgesOnCell
+      real (kind=RKIND), dimension(:), pointer :: xCell, dvEdge
+      integer, pointer :: nCells, nVertLevels
+      integer :: iCell, jCell, iNeighbor, n_damage_downstream
+      real(kind=RKIND) :: damage_downstream, damage_tmp, cellCalvingFrontLength, cellCalvingFrontHeight, seconds, R, A01, A02, Q1, Q2
+      real(kind=RKIND), dimension(:,:), pointer :: uReconstructX, uReconstructY
+      character (len=StrKIND), pointer :: xtime
```

Comment:
I don't think xtime is used in this routine - this line can be deleted.

Reply:
deleted